README.md

```diff
@@ -26,7 +26,13 @@ dashboard repository. You need to then copy the contents of `_site/` into the
 
 ## Usage
 
-To get help, you can run the container with no arguments.
+To get help, you can run the container with the `--help` argument, which will
+print the usage and display [examples](examples.md).
+
+> [!NOTE]
+>
+> The default working directory of the image is in `/site`. This is where
+> we expect the dashboard source and subfolders to live.
```

Comment:
```suggestion
> The default working directory inside the container, set in the image using `WORKDIR`, is `/site`. This is where we expect the dashboard source and its subfolders to live, and where all commands are executed unless a different working directory is specified
```
Just adding an extra breadcrumb with `WORKDIR` and being a bit more explicit re container vs image